### PR TITLE
change repository URL of all addons

### LINKED ISSUE #4705: Set addon package.json repository to sub-folder containing the addon
Context: https://github.com/xtermjs/xterm.js/pull/4519#discussion_r1299174684

eg. this should point at https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-fit

https://github.com/xtermjs/xterm.js/blob/6f0a5c9f0c1ad3e25ae3cf62f758ac99695a7210/addons/xterm-addon-fit/package.json#L10

### PR DESCRIPTION
**Pull request by ujjwalpandey922**
Fixes #4705

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Please adjust to your needs (see https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml)
+# and commit this file to your remote git repository to share the goodness with others.
+
+# Learn more from ready-to-use templates: https://www.gitpod.io/docs/introduction/getting-started/quickstart
+
+tasks:
+  - init: yarn install && yarn run build
+    command: yarn run start
+
+
```
Comment (Tyriar) [inline .gitpod.yml:11]: Can you remove this file? We like to keep the number of files in the root as small as possible.

**Review by Tyriar — CHANGES_REQUESTED** [review]
LGTM, just the one change

File: .gitpod.yml (hunk 1 above)
Comment (ujjwalpandey922) [reply .gitpod.yml]: Yes Done...

**Review by Tyriar — APPROVED** [review]

**Comment by Tyriar** [comment]
Thanks!

**Pull request merged**